protocols/IPM_Medical_Group/mass_spec_sample_prep.ot2.py

```diff
@@ -0,0 +1,81 @@
+from opentrons import labware, instruments
+
+# customization
```

Comment:
Note, if this were to be put on the library w/ customization fields would need to use `def run_custom_protocol()`. For now it is fine to keep as this since we want to get it to them ASAP and need to email out.